components/CreateProfile.js

```diff
@@ -315,6 +361,30 @@ const CreateProfile = ({
               )}
             </StyledInputField>
           </Box>
+          <Box mb={24}>
+            <StyledInputField
+              name="orgLegalName"
+              htmlFor="orgLegalName"
+              required={false}
+              label={formatMessage(messages.legalName)}
+              mt={3}
+              isPrivate
+              hint={
+                <FormattedMessage
+                  id="legalName.description"
+                  defaultMessage="The legal name is private and shared with the hosts for donation receipts, tax forms and when you submit an expense. This name is not displayed publicly and it must be your legal name."
+                />
+              }
```

Comment:
Let's not repeat this, we already describe the field above

![image](https://user-images.githubusercontent.com/1556356/142415775-dff75c69-67e9-496b-8403-dc629ecd8008.png)


```suggestion
```